src/components/Nav/nav.css

```diff
@@ -0,0 +1,23 @@
+.nav {
+  display: flex;
+  font-size: 24px;
+  box-shadow: 0px -5px 14px 4px rgba(0, 0, 0, 0.2);
+}
+
+.nav__item {
+  padding: 30px 40px;
+}
+
+.nav__item:nth-child(2) {
```

Comment:
Is this to get the separation between logo and the nav right side links ?

Reply:
Yes sir